msteams-platform/sbs-gs-commandbot.yml

```diff
@@ -260,7 +260,7 @@ items:
 
         :::image type="content" source="~/assets/images/sbs-command-bot/provision-cloud.png" alt-text="Screenshot showing the provisioning commands":::
 
-    1. Select an existing or create new resource group.
+    1. Select an existing or create new resource group.  For more information, see [Create resource group](/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups).
```

Comment:
Extra space.
Select an existing what? Mention resource group. The sentence doesn't read well.

Reply:
Extra space is removed.